server/server.go

```diff
@@ -529,20 +529,28 @@ func (s *Server) wipeIfAppropriate(ctx context.Context) {
 		}
 	}
 
+	wipeResourceCleaner := resourcestore.NewResourceCleaner(ctx)
 	if shouldWipeContainers {
 		for _, sb := range s.ContainerServer.ListSandboxes() {
+			sb := sb
 			cleanupFunc := func() error {
 				if err := s.stopPodSandbox(ctx, sb); err != nil {
 					return err
 				}
 				return s.removePodSandbox(ctx, sb)
 			}
 			if err := cleanupFunc(); err != nil {
-				log.Warnf(ctx, "Failed to cleanup pod %s: %v", sb.ID(), err)
+				log.Warnf(ctx, "Failed to cleanup pod %s once, will try again: %v", sb.ID(), err)
```

Comment:
```suggestion
				log.Warnf(ctx, "Failed to cleanup pod %s (will retry): %v", sb.ID(), err)
```
and maybe demote from warning to debug?